React/src/components/products/ProductGrid.styles.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+export const Grid = styled.div`
+  display: grid;
+  gap: 40px 24px;
+
+  @media ${props => props.theme.mobile} {
+    gap: 40px 16px;
+  }
+
+`;
+
+export const Card = styled.div`
+  background: white;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+`;
```

Comment:
상품 목록은 시맨틱하게 ul/li로 표현하는 건 어떨까요? 접근성 뿐만 아니라, 직관적으로 태그의 목적을 알 수 있어서 협업 시에도 좋아요.

```suggestion
// Grid를 ul, Card를 li로 변경
export const Grid = styled.ul`...`;
export const Card = styled.li`...`;
```